src/shared/components/RadioButton/RadioButton.style.ts

```diff
@@ -16,70 +15,6 @@ type RadioButtonCaptionProps = {
   error?: boolean
 }
 
-export const Input = styled.input`
-  margin: auto;
-  opacity: 0;
-  cursor: ${(props) => (props.disabled ? 'not-allowed' : 'pointer')};
-`
-
-const colorFromProps = ({ error, checked, disabled }: RadioButtonStyleProps) => {
-  if (disabled) {
-    return css`
-      opacity: 0.5;
-      background-color: ${checked ? colors.blue[50] : colors.gray[400]};
-      border: ${checked ? `2px solid ${colors.gray[400]}` : `1px solid ${colors.gray[200]}`};
-      background-clip: ${checked ? 'content-box' : 'unset'};
-
-      &::before {
-        top: 0;
-        bottom: 0;
-        left: 0;
-        right: 0;
-        background-color: ${colors.transparent};
-      }
-    `
-  } else if (error) {
-    return css`
-      background-color: ${checked ? colors.error : colors.transparent};
-      border: ${checked ? `2px solid ${colors.error}` : `1px solid ${colors.error}`};
-    `
-  } else {
-    return css`
-      border: ${checked ? `2px solid ${colors.blue[500]}` : `1px solid ${colors.gray[200]}`};
-      background-color: ${checked ? colors.blue[50] : colors.transparent};
-
-      &::before {
-        transition: background-color 200ms ${transitions.easing};
-      }
-
-      &:hover {
-        border: ${checked ? `2px solid ${colors.blue[500]}` : `1px solid ${colors.gray[200]}`};
-
-        &::before {
-          background-color: ${colors.transparentPrimary[12]};
-        }
-      }
-
-      &:focus {
-        border-color: ${checked ? colors.transparent : colors.gray[700]};
-
-        &::before {
-          background-color: ${checked ? colors.blue[200] : colors.gray[50]};
-        }
-      }
-
-      &:active {
-        border: 1px solid ${colors.gray[50]};
-        padding: ${checked && '4px'};
-
-        &::before {
-          background-color: ${colors.transparentPrimary[6]};
-        }
-      }
-    `
-  }
-}
-
 export const RadioButtonLabel = styled.label<RadioButtonStyleProps>`
   --radio-button-size: 18px;
   --radio-button-container-size: ${sizes(6)};
```

Comment:
`--radio-button-size` isn't used anymore, let's remove it

`--radio-button-container-size` is used for `RadioButtonContainer` but I don't think we really need it? Tbh we shouldn't need `RadioButtonContainer` at all